lib/vfs/src/lib.rs

```diff
@@ -38,6 +38,7 @@ pub mod pipe;
 pub mod static_fs;
 #[cfg(feature = "webc-fs")]
 pub mod webc_fs;
+mod webc_volume_fs;
```

Comment:
I've added a `#[cfg]`. I'm not going to put the webc-related code under a common submodule because the `webc_fs` will be deprecated/removed down the track.